build.gradle

```diff
@@ -189,6 +189,7 @@ project(':ambry-api') {
         compile "org.codehaus.jackson:jackson-core-asl:$jacksonVersion"
         compile "org.codehaus.jackson:jackson-mapper-asl:$jacksonVersion"
         compile "io.netty:netty-buffer:$nettyVersion"
+        compile "io.netty:netty-all:$nettyVersion"
```

Comment:
Agree with Ze it would be best to avoid.  Also, doesn't netty-all include netty-buffer?

Reply:
reverted this change as I have removed the dependency on netty-all from ambry-api anymore.